salt/gobblin/init.sls

```diff
@@ -61,6 +82,14 @@ gobblin-update_gobblin_reference_configuration_file:
     - require:
       - archive: gobblin-dl-and-extract
 
+gobblin-update_gobblin_mapreduce_sh_file:
+  file.replace:
+    - name: {{ gobblin_real_dir }}/gobblin-dist/bin/gobblin-mapreduce.sh
+    - pattern: 'data-2.6.0.jar'
+    - repl: 'data-11.0.0.jar'
```

Comment:
Is this still 11 if we build some other version of Gobblin?

Reply:
Actually, the incorrect name in map-reduce driver script (gobblin-mapreduce.sh) is bug in upstream apache incubator-gobblin. 
I have observed the same implementation in 0.10.0 and 0.11.0 version. 

This is temporary fix and looks good to me because 
- in case it is corrected in upstream, the map-reduce driver script will not be changed.
- in future the data-x.x.x.jar version will not go back from 11.0.0 to 2.6.0.
 
Are you suggesting to put the name in pillar by corresponding the data-x.x.x.jar appropriate version  to the upstream gobblin version ??

Reply:
My assumption was the 11 related to version 11 of Gobblin as so this would change when it goes to 12. If that's the case, we should parameterize it accordingly. However, if it's a coincidence that it's at 11 and not expected to change when we go to 12 then I would tend to agree that it's an implementation detail that the code will need to track.

Reply:
The version in map-reduce driver script should be correct.
Reason we are using gobblin 0.11.0 ( latest ) now, which has bug, This temporary fix should work.

Now the issue is fixed in apache incubator-gobblin, but not available in 0.10.0 and 0.11.0 release tar files.
**Commit Details** - https://github.com/apache/incubator-gobblin/commit/4c06d3be7e3bab6e1a0e9fa7c9df8ac7e7274a99#diff-b86d7660ecf7881c2d841c4bd83b709f

Please suggest.